lms/djangoapps/course_structure_api/v0/views.py

```diff
@@ -77,34 +77,52 @@ class CourseList(CourseViewMixin, ListAPIView):
     """
     **Use Case**
 
-        CourseList returns paginated list of courses in the edX Platform. The list can be filtered by course_id.
+        Get a paginated list of courses in the edX Platform. 
 
-    **Example Request**
+        The list can be filtered by course_id.
+
+        Each page in the list can contain up to 10 courses.
+
+    **Example Requests**
 
           GET /api/course_structure/v0/courses/
+
           GET /api/course_structure/v0/courses/?course_id={course_id1},{course_id2}
```

Comment:
This might be an old URL.  It should be something like: `/api/course_structure/v0/course_structures/{course_id}`.